### PR TITLE
Adding qte and BibTeX formats (04-halstead-complexity)

### PR DESCRIPTION
**Pull request by awggeez**

### CODE REVIEW

File: 04-halstead-complexity/04-halstead-complexity.tex [hunk 1]
```diff
@@ -92,9 +92,11 @@
 \end{itemize}
 }
 
-\pitch{
-\pptQuote{bill-curtis.jpg}{In studying some error data provided us by Rome Air Development Center, Phil Milliman and I found Halstead's metric a remarkably accurate predictor of delivered bugs in a system developed with modern programming practices and tools}{\textit{Program Complexity and Software Errors: \newline A Front End for Reliability}, \newline Bill Curtis (father of SEI CMM), 1979}
-}
+\qte
+  [Bill Curtis]
+  {{bill-curtis.jpg}
```
Comment (yegor256) [inline 04-halstead-complexity/04-halstead-complexity.tex:97]: @awggeez it's a typo: extra opening curled bracket

File: main.bib [hunk 2]
```diff
@@ -1148,4 +1148,10 @@ @article{bieman1995cohesion
...
   year={1995},
   publisher={ACM New York, NY, USA}
-}
\ No newline at end of file
+}
+
+@article{curtis1979,
+  author = {Curtis, Bill},
+  title = {Program Complexity and Software Errors: A Front End for Reliability},
+  year = {1979}
```
Comment (yegor256) [inline main.bib:1156]: @awggeez if it's an `@article`, let's add its `journal` and also `doi`

File: 04-halstead-complexity/04-halstead-complexity.tex (hunk 1 above)
Comment (awggeez) [reply 04-halstead-complexity/04-halstead-complexity.tex]: typo corrected

File: main.bib (hunk 2 above)
Comment (awggeez) [reply main.bib]: the record type was changed and the corresponding fields were added

**Comment by yegor256** [comment]
@awggeez thanks!

**Pull request merged**